di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -0,0 +1,109 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Transform: AWS::Serverless-2016-10-31
+Globals:
+  Function:
+    Timeout: 30
+
+Description: >-
+  This creates the lambda functions to stub CIMIT lambdas for getContraIndicators, getContraIndicatorCredential, 
+  putContraIndicators and postMitigations.
+
+Parameters:
+  VpcStackName:
+    Description: >
+      The name of the stack that defines the VPC in which this container will
+      run.
+    Type: String
+  Environment:
+    Description: The name of the environment to deploy to.
+    Type: String
+    AllowedPattern: ((production)|(build)|(dev.*))
+
+Resources:
+  # lambda to replicate cimit getContraIndicators
+  GetContraIndicatorsLambda:
```

Comment:
Rename resource as `GetContraIndicatorsFunction` for consistency with core-back

Reply:
done